packages/sanity/src/core/form/studio/inputs/client-adapters/reference.ts

```diff
@@ -100,11 +100,7 @@ export function getReferenceInfo(
             } as const)
           }
 
-          const previewPaths = [
-            ...(getPreviewPaths(refSchemaType?.preview) || []),
-            ['_updatedAt'],
-            ['_createdAt'],
-          ]
+          const previewPaths = [...(getPreviewPaths(refSchemaType?.preview) || [])]
```

Comment:
Can be simplified to:
```suggestion
          const previewPaths = getPreviewPaths(refSchemaType?.preview) || []
```